cypress-shared/support/location-availability/shipping-policy.graphql.js

```diff
@@ -0,0 +1,51 @@
+import { FAIL_ON_STATUS_CODE } from '../common/constants'
+
+const config = Cypress.env()
+
+// Constants
+const { vtex } = config.base
+
+function commonGraphlValidation(response) {
```

Comment:
Can u pls add this validation too
 expect(response.body).to.not.equal('OK')
 expect(response.body).to.not.have.own.property('errors')